forest/sycamore/survey_config.py

```diff
@@ -201,7 +201,41 @@ def survey_submits(study_dir, config_path, time_start, time_end, beiwe_ids, agg,
     submit_lines_summary.columns = ['survey id', 'beiwe_id', 'num_surveys', 'num_complete_surveys', 'avg_time_to_submit']
     
     return submit_lines3.sort_values(['survey id', 'beiwe_id']).drop_duplicates(), submit_lines_summary
+
+
+
+def survey_submits_no_config(study_dir, agg):
+    '''
+    Alternative function for getting the survey completions (doesn't have expected times of surveys)
+    Args:
+        study_dir(str):
+            File path to study data
+        agg(DataFrame):
+            Output of aggregate_surveys_config
+    '''
+    
+    tmp = agg.copy()
```

Comment:
When this is used in the main function the agg object is created once and shared between many functions ([here](https://github.com/onnela-lab/forest/blob/038a3fd2d0aba93a8ac1dd76be7a8d4e341bc609/forest/sycamore/sycamore_main.py#L26)). I make a copy to avoid altering the agg object between the function calls.

Reply:
Thanks, I see. Do you need to include all the data from `agg` in the return value (`tmp`)?

Reply:
Hi Ilya - I've made some updates so that I am no longer using the copy of `agg`. Let me know if you have any questions.